route.rb

```diff
@@ -1 +1,19 @@
+class Route
+  attr_reader :stations
 
+  def initialize(start, finish)
+    @stations = [start, finish]
+  end
+
+  def add_station(station)
+    @stations.insert(-2, station)
+  end
+
+  def delete_station(station)
+    @stations.delete_at(station) if @stations.first != station && @stations.last != station
```

Comment:
```suggestion
    @stations.delete(station) if @stations.first != station && @stations.last != station
```
`delete_at` работает с индексами, а мы договаривались, что в аргументах приходит объект